pkg/ovssubnet/controller/multitenant/multitenant.go

```diff
@@ -77,27 +77,32 @@ func generateCookie(ip string) string {
 }
 
 func (c *FlowController) AddServiceOFRules(netID uint, IP string, protocol api.ServiceProtocol, port uint) error {
-	rule := generateServiceRule(netID, IP, protocol, port)
+	rule := generateAddServiceRule(netID, IP, protocol, port)
 	o, e := exec.Command("ovs-ofctl", "-O", "OpenFlow13", "add-flow", "br0", rule).CombinedOutput()
 	log.Infof("Output of adding %s: %s (%v)", rule, o, e)
 	return e
 }
 
 func (c *FlowController) DelServiceOFRules(netID uint, IP string, protocol api.ServiceProtocol, port uint) error {
-	rule := generateServiceRule(netID, IP, protocol, port)
+	rule := generateDelServiceRule(IP, protocol, port)
 	o, e := exec.Command("ovs-ofctl", "-O", "OpenFlow13", "del-flows", "br0", rule).CombinedOutput()
 	log.Infof("Output of deleting %s: %s (%v)", rule, o, e)
 	return e
 }
 
-func generateServiceRule(netID uint, IP string, protocol api.ServiceProtocol, port uint) string {
+func generateAddServiceRule(netID uint, IP string, protocol api.ServiceProtocol, port uint) string {
+	baseRule := generateDelServiceRule(IP, protocol, port)
```

Comment:
nit, generateAddServiceRule() calling generateDelServiceRule() is unintuitive. Prefer renaming generateDelServiceRule to generateBaseServiceRule or some other name.